extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -693,27 +710,27 @@ static public function setVarById( $cv_variable_id, $city_id, $value, $reason=nu
 					 * ...so get the other variable
 					 */
 					if ( $variable->cv_name === "wgServer" ) {
-						$tmp = self::getVarValueByName( "wgScriptPath", $city_id );
+						$tmp = static::getVarValueByName( "wgScriptPath", $city_id );
 						$server = is_null( $value ) ? "" : $value;
 						$script_path = is_null( $tmp ) ? "/" : $tmp . "/";
 					}
 					else {
-						$tmp = self::getVarValueByName( "wgServer", $city_id );
+						$tmp = static::getVarValueByName( "wgServer", $city_id );
 						$server = is_null( $tmp ) ? "" : $tmp;
 						$script_path = is_null( $value ) ? "/" : $value . "/";
 					}
 					$city_url = $server . $script_path;
 					try {
 						$dbw->update(
-							self::table("city_list"),
+							static::table("city_list"),
 							[ "city_url" => $city_url ],
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "city_url" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)